### ISSUE TITLE
Parser ignores -RlsGroup if it includes non ascii letters

### ISSUE THREAD

**Issue by Shiginn** [Sonarr/Sonarr#8341]
### Is there an existing issue for this?

- [x] I have searched the existing open and closed issues

### Current Behavior

If the release group in the release title has a non ascii letter (such as é, à, ß or ð), the release group will not be detected by the parser. This only happens when release group is at the end of the title.
For example, on `Series Name S01E05 1080p WEB H264 AAC -abcdé` the release group is not detected, but on `Series Name S01E05 1080p WEB H264 AAC -abcde` and `[abcdé] Series Name S01E05 1080p WEB H264 AAC` the release group is detected as `abcde` and `abcdé`.

### Expected Behavior

The release group should be detected by the parser.

### Steps To Reproduce

1. Open Test Parsing
2. Input the release title

### Environment

```markdown
- OS: Windows 11 24H2
- Sonarr: 4.0.16.2944
- Docker Install: No
- Using Reverse Proxy: No
- Browser: Firefox 143 (or any)
- Database: Sqlite 3.36.0
```

### What branch are you running?

Main

### Trace Logs?

.

**Comment by markus101** [comment]
Please provide some actual release names where you're seeing this, instead of the ones provided.

**Comment by Shiginn** [comment]
I have updated the initial message with the release titles that I noticed the issue on.
